styles/main.css

```diff
@@ -66,32 +70,22 @@ header>nav>ul {
 /*mian*/
 main{
     margin-bottom: 6%;
-}
-main::after{
-    content: '';
-    display: block;
-    clear: both;
+
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 2em;

```

Comment:
The main element needs to have a grid display and 2 equally-spaced columns with a column gap